src/sql/workbench/contrib/dashboard/browser/core/dashboardPage.component.ts

```diff
@@ -239,12 +239,29 @@ export abstract class DashboardPage extends AngularDisposable implements IConfig
 
 		// get extension actions contributed to the page's toolbar
 		const contributedTasks = this.getContributedTasks(tabId);
+		if (content.length > 0 && contributedTasks.length > 0) {
```

Comment:
Do you need the `contributedTasks` check given that content is generated from the contributed tasks - so if there is anything in content shouldn't that always mean contributedTasks also has items?